### PR TITLE
Michael/new dockerfiles

### PR DESCRIPTION
**Pull request by michaelroytman**

### CODE REVIEW

**Comment by michaelroytman** [comment]
@edx/devops

File: util/README.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
+# How to add Dockerfiles to configuration file
+
+The script that handles distributing build jobs across Travis CI shards relies on the parsefiles_config YAML file. This file contains a mapping from each application that has a Dockerfile to its corresponding weight/rank. The rank refers to the approximate running time of the a Travis Docker build for that application's Dockerfile. When adding a new Dockerfile to the configuration repository, this configuration file needs to be manually updated in order to ensure that that Dockerfile is also built.
```
Comment (jibsheet) [inline util/README.md]: s/the a Travis/a Travis/

File: util/check_dockerfile_coverage.py [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+import logging
+import sys
+
+TRAVIS_BUILD_DIR = os.environ.get("TRAVIS_BUILD_DIR")
+CONFIG_FILE_PATH = pathlib2.Path(TRAVIS_BUILD_DIR, "util", "parsefiles_config.yml")
+LOGGER = logging.getLogger(__name__)
+
+def check_coverage(containers):
+    # open config file containing container weights
+    config_file_path = pathlib2.Path(CONFIG_FILE_PATH)
```
Comment (jibsheet) [inline util/check_dockerfile_coverage.py]: I think this code is mostly identical to balancecontainers.py? up until where we calculate uncovered.
Is there a simple refactor we can do to share that code?

**Comment by jibsheet** [comment]
1 typo one medium question.
Once this lands, I'll fix tools-jenkins to run properly.

**Comment by fredsmith** [comment]
👍

**Pull request merged**